Source/ProceduralFairings/FairingDecoupler.cs

```diff
@@ -112,8 +112,22 @@ public void FixedUpdate ()
 
                     if (tr)
                     {
-                        part.Rigidbody.AddForce (tr.TransformDirection (forceVector) * Mathf.Lerp (ejectionLowDv, ejectionDv, ejectionPower), ForceMode.VelocityChange);
-                        part.Rigidbody.AddTorque (tr.TransformDirection (torqueVector) * Mathf.Lerp (ejectionLowTorque, ejectionTorque, torqueAmount), ForceMode.VelocityChange);
+                        Vector3d Δv = tr.TransformDirection(forceVector) * Mathf.Lerp(ejectionLowDv, ejectionDv, ejectionPower);
+                        part.AddImpulse(part.mass * Δv);
+
+
```

Comment:
Maybe drop one if these empty lines?  One line good, two lines bad.

Reply:
I agree. Done, fixed in the latest push.